src/openfermion/utils/_remove_symmetry_qubits.py

```diff
@@ -0,0 +1,118 @@
+""" Module to remove two qubits from the problem space
+    using conservation of electron number and conservation
+    of electron spin. As described in arXiv:1701.08213.
+"""
+
+
+import openfermion
+from openfermion.ops import FermionOperator, QubitOperator
+from openfermion.transforms import get_fermion_operator, bravyi_kitaev_tree
```

Comment:
our style guide mandates alphabetical imports. So bravyi_kitaev_tree should come before get_fermion_operator.